### PR TITLE
refactor: error and feature flag for env function

### PR DESCRIPTION
**Pull request by yuqizhou77**
- Error message if processing function valided.
- Add feature flag.

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -522,6 +522,18 @@
   "core.common.ErrorFetchApiSpec": "Your OpenAPI description document should be accessible without authentication, otherwise download and start from a local copy.",
   "core.common.SendingApiRequest": "Sending API request: %s. Request body: %s",
   "core.common.ReceiveApiResponse": "Received API response: %s.",
+  "core.envFunc.unsupportedFile.errorLog": "\"%s\" is not a valid file. Supported format: %s.",
+  "core.envFunc.unsupportedFile.errorMessage": "Invalid file. Check output for details.",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:526]: Do we want to say 'output panel' here?

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -522,6 +522,18 @@
...
+  "core.envFunc.unsupportedFile.errorMessage.vsc": "Invalid file. Check [Output panel](command:fx-extension.showOutputChannel) for details.",
+  "core.envFunc.unsupportedFunction.errorLog": "\"%s\" is not a valid function. Supported function: %s.",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:528]: "\"%s\" is an invalid function. Supported function: %s.",

File: packages/fx-core/resource/package.nls.json [hunk 3]
```diff
@@ -522,6 +522,18 @@
...
+  "core.envFunc.unsupportedFunction.errorMessage": "Invalid function. Check output for details.",
+  "core.envFunc.unsupportedFunction.errorMessage.vsc": "Invalid function. Check [Output panel](command:fx-extension.showOutputChannel) for details.",
+  "core.envFunc.invalidFunctionParameter.errorLog": "The parameter \"%s\" of function \"%s\" is invalid. Please provide a valid file path wrapped by '' or an environment variable name in \"${{}}\" format.",
+  "core.envFunc.invalidFunctionParameter.errorMessage": "Invalid parameter of function \"%s\". Check output for details.",
+  "core.envFunc.invalidFunctionParameter.errorMessage.vsc": "Invalid parameter of function \"%s\". Check [Output panel](command:fx-extension.showOutputChannel) for details.",
+  "core.envFunc.readFile.errorLog": "Failed to read from \"%s\" due to \"%s\".",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:534]: Proposed: "Unable to read from \"%s\" due to \"%s\"."

File: packages/fx-core/resource/package.nls.json [hunk 4]
```diff
@@ -522,6 +522,18 @@
...
+  "core.envFunc.readFile.errorMessage": "Failed to read from \"%s\". Check output for details.",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:535]: Proposed: "Unable to read from \"%s\". Check output for details."

File: packages/fx-core/resource/package.nls.json [hunk 5]
```diff
@@ -522,6 +522,18 @@
...
+  "core.envFunc.readFile.errorMessage.vsc": "Failed to read from \"%s\". Check [Output panel](command:fx-extension.showOutputChannel) for details.",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:536]: Proposed: "Unable to read from \"%s\". Check [Output panel](command:fx-extension.showOutputChannel) for details."

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (yuqizhou77) [reply packages/fx-core/resource/package.nls.json]: I have removed "Check output for details." as TTK will always add such sentence at the end of the error if running in VS.

File: packages/fx-core/resource/package.nls.json (hunk 4 above)
Comment (yuqizhou77) [reply packages/fx-core/resource/package.nls.json]: Thanks, updated

**Review by jayzhang — APPROVED** [review]

**Review by supkasar — APPROVED** [review]

**Pull request merged**